cpp/oneapi/dal/algo/kmeans/test/batch.cpp

```diff
@@ -318,22 +318,33 @@ TEMPLATE_LIST_TEST_M(kmeans_batch_test,
     }
 }
 
+#ifdef ONEDAL_DATA_PARALLEL
```

Comment:
I guess this test can be launched only in dpcpp backend, like this ifdef defined always. Not sure that it makes sense to have this #ifdef here

Reply:
Testing on CPU fails without this #ifdef. I will leave it as is.